packages/salesforcedx-vscode-apex/test/jest/views/settings/configurations.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+
+describe('Should synchronize init jobs in server by default', () => {
+  const sandbox = sinon.createSandbox();
+  beforeEach(() => {
+    sandbox.stub(vscode.workspace, 'getConfiguration').returns({
+      get: () => true
+    });
```

Comment:
Testing default values of vscode setting is outside the scope of what a unit test should accomplish.